packages/module/src/TagCount/TagCount.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import clsx from 'clsx';
+import { Button, ButtonProps, Icon } from '@patternfly/react-core';
+import { TagIcon } from '@patternfly/react-icons';
+import { createUseStyles } from 'react-jss'
+
+const useStyles = createUseStyles({
+  buttonTagCount: {
+    color: 'var(--pf-v5-global--icon--Color--light)',
+    display: 'flex',
+    alignItems: 'center'
+  },
+
+  tagText: {
+    marginLeft: 10
+  }
+});
+
+export interface TagCountProps extends ButtonProps {
+  count?: number;
+  onTagClick?: React.MouseEventHandler<HTMLButtonElement>;
```

Comment:
Can we keep the `onClick` interface? The callback goes to the button anyway.

Reply:
Done